packages/design-system-react-native/src/index.ts

```diff
@@ -1,5 +1,5 @@
 import { withThemeProvider } from '@metamask/design-system-twrnc-preset';
 
-import ButtonComponent from './components/Button';
+import TextComponent from './components/Text';
 
-export const Button = withThemeProvider(ButtonComponent);
+export const Text = withThemeProvider(TextComponent);
```

Comment:
I think we may also want to export the `Text` types e.g. `TextProps` so they can be extended in custom app components